content/kubermatic/master/installation/add_seed_cluster/_index.en.md

```diff
@@ -0,0 +1,211 @@
++++
+title = "Add Seed Cluster"
+date = 2018-08-09T12:07:15+02:00
+weight = 30
+pre = "<b></b>"
++++
+
+This document describes how a new seed cluster can be added to an existing Kubermatic master cluster.
+
+### 1. Install Kubernetes cluster
```

Comment:
There we should explain, with a drawing, what is meant by a seed cluster and how potential setups could look like. And that also a the cluster from `Install Kubermatic` could be "used as an seed cluster"
See the drawing of our pitch slides https://docs.google.com/presentation/d/1fgakLIpHMtAXPDgGfSam4Dpzf75XspELITEVt5KAt_o/edit?pli=1#slide=id.g7071afd60d_2_1165